content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Exporters
+weight: 50
+---
+
+In order to visualize and analyze your traces, you will need to export them to a
+backend such as [Jaeger](https://www.jaegertracing.io/) or
+[Zipkin](https://zipkin.io/). OpenTelemetry Java provides exporters for some
+common open source backends.
+
+If you use the Javaagent for
+[automatic instrumentation](/docs/instrumentation/java/automatic) you can learn
+how to setup exporters following the
+[Agent Configuration Guide](/docs/instrumentation/java/automatic/agent-config)
+
+For [manual instrumentation](/docs/instrumentation/java/manual), you will find
+some introductions below on how to set up backends and the matching exporters.
+
+## OTLP endpoint
+
+To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to use `opentelemetry-exporter-otlp`:
+
+<!-- prettier-ignore-start -->
+
+{{< tabpane text=true >}}
+{{% tab Gradle %}}
+
+```kotlin
+dependencies {
+    implementation 'io.opentelemetry:opentelemetry-exporter-otlp:{{% param javaVersion %}}'
+}
+```
+
+{{% /tab %}}
+{{% tab Maven %}}
+
+```xml
+<project>
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-otlp</artifactId>
+        </dependency>
+    </dependencies>
+</project>
+```
+
+{{< /tab >}}
+{{< /tabpane>}}
+
+Next, configure the exporter to point at an OTLP endpoint.
+
+If you use [SDK auto-configuration](/docs/instrumentation/java/manual/#automatic-configuration)
```

Comment:
Are you going to break out a page dedicated to auto configuration?

Reply:
Eventually if needed, yes, but maybe not in this PR.